src/app/products/empty-cart/empty-cart.component.spec.ts

```diff
@@ -10,7 +10,7 @@ describe('EmptyCartComponent', () => {
     await TestBed.configureTestingModule({
```

Comment:
It's just a UI so the test was to make sure the UI is rendered and I think that's enough

Reply:
You are showing certain elements, test whether they exist or not. Because they are the essential of that simple component. 
Reason: because if you are working in a team and someone mistakenly takes something out (essential stuff), the test case will catch that. Or it can help someone understand the whole either behind the component